src/main/java/org/folio/circulation/rules/Text2Drools.java

```diff
@@ -345,6 +345,8 @@ private String policyMatchString(PolicyContext policy) {
       policyTypeName = "request";
     } else if (policy.POLICY_TYPE().toString().equals("n")) {
       policyTypeName = "notice";
+    } else if (policy.POLICY_TYPE().toString().equals("o")) {
+      policyTypeName = "overdue";
```

Comment:
Can you use a switch for readability?

    switch (policy.POLICY_TYPE().toString()) {
    case "l": return "    match.loanPolicyId = ";
    case "r": return "    match.requestPolicyId = ";
    case "n": return "    match.noticePolicyId = ";
    case "o": return "    match.overduePolicyId = ";
    default: throw new IllegalArgumentException("Unknown policy type: " + policy.POLICY_TYPE().toString());
    }

Reply:
Done.